enterprise/app/review/view_pull_request.tsx

```diff
@@ -691,3 +694,24 @@ export default class ViewPullRequestComponent extends React.Component<ViewPullRe
     );
   }
 }
+
+const FILES_TO_PREFETCH = 3;
```

Comment:
nit: should this go at the top of the file?

Reply:
sure, done.